pontoon/contributors/static/js/profile.js

```diff
@@ -37,7 +37,12 @@ var Pontoon = (function (my) {
         var ctx = chart[0].getContext('2d');
 
         var gradient = ctx.createLinearGradient(0, 0, 0, 160);
-        gradient.addColorStop(0, '#7BC87633');
+        gradient.addColorStop(
+          0,
+          getComputedStyle(document.documentElement)
+            .getPropertyValue('--dark-green')
+            .trim(),
```

Comment:
We should declare `style = getComputedStyle(document.documentElement);` only once per file, and then reference colors just by using `style.getPropertyValue('--dark-green')` and the likes.

We should drop [`trim()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trim). Is doesn't make a functional difference, does it?